builtin/logical/transit/backend_test.go

```diff
@@ -587,15 +602,18 @@ func TestPolicyFuzzing(t *testing.T) {
 		}
 		fd := &framework.FieldData{}
 
+		var chosenFunc, chosenKey string
+
+		//t.Errorf("Starting %d", id)
 		for {
 			// Stop after 10 seconds
 			if time.Now().Sub(startTime) > 10*time.Second {
 				return
 			}
 
 			// Pick a function and a key
-			chosenFunc := funcs[rand.Int()%len(funcs)]
-			chosenKey := keys[rand.Int()%len(keys)]
+			chosenFunc = funcs[rand.Int()%len(funcs)]
```

Comment:
Can these `rand.Int()%len(funcs)` calls be changed to `rand.Intn(len(funcs))` instead?  There's a power-of-two optimization in `Intn()`, `Int31n()`, and `Int63n()`.  For reference:  https://golang.org/src/math/rand/rand.go?s=6757:6777#L77